src/DUWOagent/lowBudgetMod.mod2g

```diff
@@ -10,11 +10,16 @@ exit=always.
 % Module for when the agent has a low budget, the agent will only continue to build student housing in places where this greatly improves the quality indicator
 module lowBudgetMod {
 	
-	% Build student housing when have it as goal.
-	if bel(functions(FunctionsList), 
-		member([Name,BuildingID,CategoryList],FunctionsList),
-		member('STUDENT',CategoryList),
-		getPolygon(10,10,200,20,Polygon)
-		), goal(goalBuildStudentHousing)  
-	then building_plan_construction(BuildingID, 1, Polygon).
+	% If we know where to build, and we need more cheap student housing,
+	% start building and update the list of multipolys.
+	% We also select a random cheap student building to build.
+	if bel(relevant_areas(0,[[Poly|Area]|Rest]),
+		cheapHousingIds(Idlist),
+		random_member(Id,Idlist))
+	then building_plan_construction(Id,4,Poly)
+		+ delete(relevant_areas(0,[[Poly|Area]|Rest]))
+		+ insert(relevant_areas(0,Rest)).
+	
+	if bel(relevant_areas(0, MultiPolygons))
+	then delete(relevant_areas(0, MultiPolygons)).
```

Comment:
Add `if true then noBudgetMod.` since we want to sell land if we dont need to build cheap student housing,